sky/serve/load_balancer.py

```diff
@@ -241,5 +259,11 @@ def run_load_balancer(controller_addr: str, load_balancer_port: int):
                         required=True,
                         default=8890,
                         help='The port where the load balancer listens to.')
+    parser.add_argument(
+        '--load-balancing-policy',
+        choices=['round_robin'],
```

Comment:
lets update this to the keys of the registry?

Reply:
Good point! 💯